userportaldatamodel/project_datapoints.py

```diff
@@ -0,0 +1,37 @@
+from . import Base
+from sqlalchemy import (
+    Integer,
+    String,
+    Column,
+    CHAR,
+)
+
+from sqlalchemy import CheckConstraint
+from sqlalchemy.dialects.postgresql import ARRAY
+import json
+
+class ProjectDataPoints(Base):
+    __tablename__ = "project_datapoints"
+
+    id = Column(Integer,primary_key=True, autoincrement=True)
+    term = Column(String)
+    value_list = Column(ARRAY(String))
+    type = Column(CHAR)
+    project_id = Column(Integer)
```

Comment:
project_id needs to be a relationship to the project table, something like this:

https://github.com/chicagopcdc/userPortalDataModel/blob/0c9e568c05575e97f3f4b3ad049d61b64ecf3d6a/userportaldatamodel/request.py#L32-L34